contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue

```diff
@@ -114,6 +134,12 @@
         default: false,
       },
     },
+    data() {
+      return {
+        reqeuestFormStrings: crossComponentTranslator(RequestForm),
```

Comment:
A small nitpick and a curiosity question: 
1. there's a typo here `reqeuest` (it's working, though! the spelling is consistent. but a nice small cleanup)
2. It seems like even though I had problems with the modal interaction previously, it was working for you with the correct text (modal-in-modal, as originally designed). I'm wondering how the strings were working? I don't see crossComponent translator used in the deleted file. It's not a problem! I'm just trying to figure it out for the sake of a comprehensive review 😄

Reply:
1. Thanks for catching this, not sure how I missed it!
2. I believe the problems you were having was because the License input was disabled. I noticed this happening when it is unable to be edited I think because it is not in its original location. The strings worked fine but they were directly in the deleted file so instead of adding them over again in `LicenseDropdown.vue`, I decided to use the crossComponent translator. I was unsure if adding the same strings to `LicenseDropdown.vue` would import the translated strings the correct way.

Reply:
okay thank you @LianaHarris360!